### PR TITLE
Adds persistent painting frames everywhere on box to totally improve RP. Poggers

### PR DESCRIPTION
**Pull request by alexkar598**
### Intent of your Pull Request

Well you see, the persistence adds immersion or something so it improves RP because that's the magic words we need to get PRs merged now i guess

### Why is this change good for the game?

It improves RP

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Painting frames will persist paintings across rounds

### What should players be aware of when it comes to the changes your PR is implementing?
Painting frames will persist paintings across rounds

### What general grouping does this PR fall under? 
Painting frames will persist paintings across rounds

# Changelog

:cl:  
rscadd: Added persistent painting frames on box. Paintings hung on them will persist across rounds(perhaps not where you left it, but somewhere on the station)
/:cl:

### CODE REVIEW

**Review by patpol4 — CHANGES_REQUESTED** [review]
Why would you have paintings in sec or med

**Comment by alexkar598** [comment]
Ambience, calm down patients, calm down the clown as hes being metaphorically raped by sec

**Comment by patpol4** [comment]
> Ambience, calm down patients, calm down the clown as hes being metaphorically raped by sec

Technically yes, but i think those are workplaces, not casual spots for paintings to be hanging in. Paintings give off more of a casual vibes, if you wanted ambience we have official posters

**Comment by alexkar598** [comment]
Posters aren't fun to look at, noone wants to look at posters because theres like a dozen and theyre the same. Paintings are virtually infinite which make them much more interesting to look at. Having art in hospitals isn't exactly uncommon either. The brig is kinda weird but theres no real reason NOT to have them there either.

**Review by patpol4 — APPROVED** [review]
Ok let's try it but I'll be removing those if it looks bad

**Comment by MegaEmpirical** [comment]
Super excited for this. Been wanting something skin to the library book system or the trophy cases for a while. Really would like to see an art exhibit to the back of the library too.

Nicely done.
